docs/02-for-app-authors/12-linter.md

```diff
@@ -375,6 +398,17 @@ The manifest has no [finish-args](https://docs.flatpak.org/en/latest/manifests.h
 
 This is only selectively allowed for console applications if needed.
 
+## finish-args-portal-own-name, finish-args-portal-talk-name
```

Comment:
Not sure if this is the best approach of you want to link these from the error, as the headline and thus anchor will have both errors.

So either set an ID https://docusaurus.io/docs/next/markdown-features/toc#heading-id's or separate these, even if they feel similar. Might also be easier for readers.

Reply:
Yea seems fine, if you want to push please do. I won't be near a PC for a while.